tests/test_core/test_annotationdb.py

```diff
@@ -0,0 +1,633 @@
+from cogent3 import load_unaligned_seqs, open_
+from cogent3.core.annotation_db import GenbankAnnotationDb, GffAnnotationDb
+from cogent3.parse.genbank import MinimalGenbankParser
+from cogent3.parse.gff import gff_parser
+import os
+
+base_path = os.path.dirname(os.path.dirname(__file__))
```

Comment:
better to use pathlib for these. Also, since they're effectively constants, they should be allcaps.

```python
import pathlib # add at top of file

DATA_DIR = pathlib.Path(__file__).parent.parent
GFF_PATH = DATA_DIR / "prok_NoLocusTags.gff"
GB_PATH = DATA_DIR / "annotated_seq.gb"
```